udaykalra/portfolio/target/portfolio-1/style.css

```diff
@@ -1,9 +1,141 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+:root {
+  --main-color: #fff;
+  --head-color: #000000;
+  --body-color: #121212;
+  --text-color: #e7e7e7;
+  --light-color: #333333;
+}
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+html,
+body {
+  margin: 0;
+  height: 100%;
+  background-color: var(--body-color);
+}
+body {
+  background: 100%;
+}
+
+div.main-content {
+  background-color: var(--body-color);
+  color: var(--main-color);
+  font-family: 'Roboto', sans-serif;
+  height: 100%;
+}
+
+.hero-image {
+  background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)),
+    url('hero.jpg');
+
+  height: 50%;
+
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+}
+
+.hero-text {
+  font-family: 'Roboto', sans-serif;
+  font-size: 40px;
+  text-align: center;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  color: white;
+}
+
+.hero-text h1 {
+  font-size: 60px;
+}
+footer {
+  position: fixed;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  font-family: 'Roboto', sans-serif;
+  text-align: center;
+  margin: 0 auto;
+  height: 5em;
+  background: var(--head-color);
+  color: var(--text-color);
+}
+div.grey {
+  background-color: var(--body-color);
+}
+div.go {
+  border-radius: 0px 0px 15px 15px;
+  background-color: var(--head-color);
+  font-family: 'Roboto', sans-serif;
+  margin: 0 auto;
+  width: 100%;
+  padding: 2em 2em;
+  text-align: center;
+}
+
+div.go a {
+  color: #fff;
+
+  font-family: 'Roboto', sans-serif;
+  text-decoration: none;
+  font-size: 30px;
+  margin: 0px 10px;
+  padding: 20px 20px;
+  position: relative;
+  z-index: 0;
+  cursor: pointer;
 }
 
-#greeting-container {
-  margin-top: 20px;
+div.animate a:before,
+div.animate a:after {
+  position: absolute;
+  left: 0px;
+  width: 100%;
+  height: 1px;
+  background-color: var(--text-color);
+  content: '';
+  opacity: 1;
+  transition: all 0.5s;
 }
+
+div.animate a:before {
+  top: 0px;
+  transform: translateY(10px);
+}
+
+div.animate a:after {
+  bottom: 0px;
+  transform: translateY(-10px);
+}
+
+div.animate a:hover:before,
+div.animate a:hover:after {
+  opacity: 0;
+  transform: translateY(0px);
+}
+
+.topRow {
+    padding-top:30px;
+
+}
+
+.textbox{
+    background: var(--light-color);
+    border: 2px solid var(--light-color);
+    border-radius: 15px;
+
+}
+.textbox h1 {
+    font-size: 50px;
+}
+.textbox p {
+    font-size: 30px;
+
```

Comment:
Removing this extra vertical white would improve consistency.

Also applies to a few rules above.

Reply:
Done.